.github/workflows/release-21_extrinsic-ordering-check-from-two.yml

```
@@ -0,0 +1,97 @@
+# This workflow performs the Extrinsic Ordering Check on demand using two reference binaries
+
+name: Release - Extrinsic API Check with reference bins
+on:
+  workflow_dispatch:
+    inputs:
+      reference_binary_url:
+        description: A url to a Linux binary for the node containing the reference runtime to test against
+        default: https://releases.parity.io/polkadot/x86_64-debian:stretch/v0.9.26/polkadot
+        required: true
+      binary_url:
+        description: A url to a Linux binary for the node containing the runtime to test
+        default: https://releases.parity.io/polkadot/x86_64-debian:stretch/v0.9.27-rc1/polkadot
+        required: true
+
+jobs:
+  check:
+    name: Run check
+    runs-on: ubuntu-latest
+    env:
+      BIN_URL: ${{github.event.inputs.binary_url}}
+      REF_URL: ${{github.event.inputs.reference_binary_url}}
+    strategy:
+      matrix:
+        chain: [polkadot, kusama, westend, rococo]
```

In case rococo fails for any reason, it is valuable to have the rest

```suggestion
    strategy:
      fail-fast: false
      matrix:
        chain: [polkadot, kusama, westend, rococo]
```